### ISSUE TITLE
Pętla

### ISSUE THREAD

**Issue by MsBasis** [Science-Club-UG-BioAI/Nightingale#11]
1. Zaimplementujcie pełną pętlę treningową z podziałem danych na train/test/val.

2. Użyjcie loss function odpowiedniego dla problemu sekwencyjnego (np. CrossEntropyLoss).

3. Monitorujcie loss, BLEU, accuracy podczas treningu.

Useful links:
- https://www.youtube.com/watch?v=v9rputAi9iE
- https://docs.pytorch.org/tutorials/beginner/introyt/trainingyt.html
- https://www.digitalocean.com/community/tutorials/training-validation-and-accuracy-in-pytorch
- https://lightning.ai/docs/torchmetrics/stable/text/bleu_score.html
